src/test/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkTaskWithSchemaTest.java

```diff
@@ -266,6 +267,39 @@ public void supportZonedDatesStringTest() {
     }
 
 
+    @Test
+    public void supportFormattedDatesStringTest() {
+        Map<String, String> props = createProps();
```

Comment:
I may be missing something here. I think it needs to be a data validation test (that our conversion is correct)?